### DISCUSSION TITLE
Should I remove up/down movement?

### DISCUSSION THREAD

**Discussion by wongjiahau** [ki-editor/ki-editor#85] [Ideas]
The main goal of this editor is not to center movement based on line/columns (in the same light as Rob Pike's idea in Structural Regexp), so by right we should not have a movement dedicated to line-wise movements.

**Comment by eko234** [comment]
I get the idea, but sometimes I'm writing like pseudocode or wathever, and some line oriented primitives are important, or something that I like about kakoune is the select inside indentation... otoh, if you plan on supporting some kind of plugins that use the ui as text, I think having a more plain text mode would be useful.
